spack/packages/dla-future/package.py

```diff
@@ -93,15 +94,18 @@ class DlaFuture(CMakePackage, CudaPackage, ROCmPackage):
             depends_on(
                 "rocblas amdgpu_target={0}".format(val), when="amdgpu_target={0}".format(val)
             )
+            depends_on("whip amdgpu_target={0}".format(val), when="amdgpu_target={0}".format(val))
```

Comment:
Nitpick: Should we start moving to f-strings, at least where we do changes? (If yes, I'd change `val` to something more self-explanatory like `gpu_target` or similar.)

Reply:
I would fix the whole package if we update the style.
Let me open an issue to address it before v0.4.0

Reply:
#1053